.github/ISSUE_TEMPLATE/usage_question.yaml

```diff
@@ -0,0 +1,66 @@
+name: Usage Question
+description: Ask a question
+labels: ["Type: usage"]
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        While we enable issues as a mechanism for new contributors and passers-by who are unfamiliar with Apache Software Foundation projects to ask questions and interact with the project, we encourage users to ask such questions on public mailing lists:
+        
+        * Development discussions: dev@arrow.apache.org (first subscribe by sending an e-mail to dev-subscribe@arrow.apache.org).
+
+        * User discussions: user@arrow.apache.org (first subscribe by sending an e-mail to user-subscribe@arrow.apache.org).
```

Comment:
Can we also link to https://arrow.apache.org/community/ where the archives can be browsed from?

Reply:
Incorporated into https://github.com/apache/arrow/pull/14688.